### PR TITLE
Bugfix/upgrade type test

### PR DESCRIPTION
**Pull request by spiderbiggen**
Fix the upgrade_types percept tests

### CODE REVIEW

**Comment by MaxGroenenboom** [comment]
Travis: [Build Status](https://travis-ci.org/spiderbiggen/tygron)

File: contextvh/src/test/java/contextvh/translators/J2UpgradeTypeTest.java [hunk 1]
```diff
@@ -24,23 +26,24 @@
...
-
+	private List<UpgradePair> pairs;
 	/**
 	 * Initialise before every test.
 	 */
 	@Before
 	public void init() {
-		translator = new J2UpgradeType();
+		translator.registerJava2ParameterTranslator(new J2UpgradeType());
+		translator.registerJava2ParameterTranslator(new J2UpgradePair());
```
Comment (MaxGroenenboom) [inline contextvh/src/test/java/contextvh/translators/J2UpgradeTypeTest.java]: Wouldn't it make more sense to mock the untested translator here? This way this test can fail if JUpgradePair fails. Doesn't really matter too much though.

**Comment by shdshdw** [comment]
What bug does this PR attempt to fix exactly?

File: contextvh/src/test/java/contextvh/translators/J2UpgradeTypeTest.java (hunk 1 above)
Comment (spiderbiggen) [inline contextvh/src/test/java/contextvh/translators/J2UpgradeTypeTest.java]: that's actually a good suggestion

**Comment by spiderbiggen** [comment]
this PR fixes a bug with the UpgradetypeTest that was noticed in PR #93 where the upgrade type test failed on not having the upgradePairs translator. This way adding the upgradepairs translator from the upgrade types constructor isn't necesarry

**Comment by shdshdw** [comment]
Could not see this causing any issues for me so approved. 
- CodeFox [CodeFox](https://github.com/codefoxnl)

**Comment by jannelie** [comment]
We approve -CrashTestDummies

**Comment by klikooo** [comment]
We approve - WhySoSerious

**Comment by dionytadema** [comment]
DaNSHaL aproves

**Comment by spiderbiggen** [comment]
5/5 :D

**Pull request merged**